packages/nns/src/types/governance_converters.ts

```diff
@@ -183,6 +184,13 @@ export interface ManageNeuron {
   command: Option<Command>;
   neuronIdOrSubaccount: Option<NeuronIdOrSubaccount>;
 }
+export interface InstallCode {
+  arg: Option<Uint8Array>;
+  wasmModule: Option<Uint8Array>;
+  skipStoppingBeforeInstalling: Option<boolean>;
+  canisterId: Option<PrincipalString>;
+  installMode: Option<number>;
```

Comment:
Just sharing in case you want to get fancy, in `@dfinity/ic-management` we use an enum for the mode

```
export enum InstallMode {
  Install,
  Reinstall,
  Upgrade,
}
```

Which maps to a variant. Don't know why in NNS they use numbers btw.

Again just saying in case interesting.

Reply:
Done.

I believe they use numbers because the candid file is generated from a proto file, in which enums work slightly different.